pkg/scheduler/core/generic_scheduler.go

```diff
@@ -195,6 +190,12 @@ func (g *genericScheduler) Schedule(state *framework.CycleState, pod *v1.Pod) (r
 
 	trace.Step("Basic checks done")
 	startPredicateEvalTime := time.Now()
+	// Run "prefilter" plugins.
```

Comment:
I have a question, why do we have to move this code? We have to accept the snapshot operation, but it seems to be redundant at times.

Reply:
because predicateMetaProducer is invoked inside findNodesThatFit, and hence it is now considered part of predicate time. predicateMetaProducer is the equivalent to PreFilter plugins.

In any case, I reverted this change because we should handle this in a separate PR.

Reply:
that is not to say this is how things should be, just that this is what the currently the predicate trace step includes.